docs/operations.md

```diff
@@ -0,0 +1,132 @@
+| operatorDeployment.operatorContainer.resources           | Resources for the operator container.                                                                                                                    | memory 4Gi, ephemeral storage 2Gi and 1 cpu                                                             |
+| operatorDeployment.additionalContainers                  | Additional containers to be added to the operator pod, e.g. sidecar.                                                                                     |                                                                                                         |
+| operatorRbac.serviceAccount.create                       | Whether to create service account for operator to use.                                                                                                   |                                                                                                         |
+| operatorRbac.clusterRole.create                          | Whether to create ClusterRole for operator to use.                                                                                                       | true                                                                                                    |
+| operatorRbac.clusterRoleBinding.create                   | Whether to create ClusterRoleBinding for operator to use.                                                                                                | true                                                                                                    |
+| operatorRbac.role.create                                 | Whether to create Role for operator to use. At least one of `clusterRole.create` or `role.create` should be enabled                                      | true                                                                                                    |
+| operatorRbac.roleBinding.create                          | Whether to create RoleBinding for operator to use. At least one of `clusterRoleBinding.create` or `roleBinding.create` should be enabled                 | true                                                                                                    |
+| operatorRbac.clusterRole.configManagement.roleName       | Role name for operator configuration management (hot property loading and leader election)                                                               | `spark-operator-config-role`                                                                            |
+| appResources.namespaces.create                           | Whether to create dedicated namespaces for Spark apps.                                                                                                   | `spark-operator-config-role-binding`                                                                    |
```

Comment:
Shall we add `clusterResources` first before adding this document? It looks a little weird because the document is missing one of the part while Apache Spark Operator supports both `SparkApp` CRD and `SparkCluster` CRD.

Reply:
Yep! Add a short field in Spark Custom Resources page to start with. Also created SPARK-49528 to better doc the template support for clusters

Reply:
Actually +1 for the point - `appResources` can be a bit misleading, since it may serve both `SparkApp` and `SparkCluster`. It was introduced to indicate this is for running Spark workload (comparing other resources created for operator deployment itself).

I shall fix this by SPARK-49623